app/build.gradle

```diff
@@ -17,6 +17,10 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         vectorDrawables.useSupportLibrary = true
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
```

Comment:
There is no need to add room. You can fetch the userFragment from the fragment manager if you cannot get the viewmodel from the backstack.

Reply:
> > @epicadk The issue solution was to Locally store profile , Saving user data locally would restrict API call totally in the Profile Fragment which will be the best solution to the issue #714 as every time the Profile fragment is opened the data is fetched locally !
> > Solution that was stated by @Priyansh-Kedia
> > This kind of API calling abuses the use of API. As the profile info is just small texts, they can be stored locally so that every time profile is opened, there is no API call
> 
> I understand however I still stick by what I said. I'll let other contributors share their thoughts on this as well.

Okay !

Reply:
> There is no need to add room. You can fetch the userFragment from the fragment manager if you cannot get the viewmodel from the backstack.

I also think same.